cmds/coredhcp/config.yml.example

```diff
@@ -156,3 +156,9 @@ server4:
         # * lease duration can be given in any format understood by go's
         # "ParseDuration": https://golang.org/pkg/time/#ParseDuration
         - range: leases.txt 10.10.10.100 10.10.10.200 60s
+
+        # optional static routes could be defined as well
+        # - staticroute: <destination>,<gateway> <destination>,<gateway>
+        # where destination should be in CIDR notation and gateway should be
+        # the IP address of the router through which the destination is reachable
```

Comment:
Maybe worth adding a mention here that local subnet routes can be specified with a gateway of `0.0.0.0` (from https://datatracker.ietf.org/doc/html/rfc3442#page-4 "Local Subnet Routes")

Reply:
I know of this option but I usually find it confusing. I updated the comments and mentioned the RFC.